python/src/equistore/operations/lstsq.py

```diff
@@ -16,6 +17,57 @@ def lstsq(X: TensorMap, Y: TensorMap, rcond, driver=None) -> TensorMap:
     for ``w``, where ``Y``, ``X`` and ``w`` are all :py:class:`TensorMap`. ``Y``
     and ``X`` must have the same ``keys``.
 
+    .. note::
+      The values of ``w`` differ from the output of numpy or torch in that they
+      are already transposed. Be aware of that if you want to manually access
+      values of ``w`` (see also the example below).
+
+    Here is an example using this function:
+
```

Comment:
I think that should fix this
```suggestion
    >>> from equistore import Labels
```